src/main/deploy/config.properties

```diff
@@ -0,0 +1,45 @@
+# the least important level at which to allow logging
+# if you're not sure, use WARN
+# see Logger.java
+LOG__VISION=VERBOSE
+LOG__POT=DEBUG
+LOG__PID=INFO
+LOG__ROBOT=INFO
+
+OI__VISION__ENABLED=false
+
+OI__VISION__POT__MIN=0
+OI__VISION__POT__MAX=100
+
+# the P, I, and D constants from PID control
+
+## present
+OI__VISION__PID__P=.03
+
+## past
+OI__VISION__PID__I=0.003
+
+## future
+OI__VISION__PID__D=0
+
+# the fastest the PID is allowed to go
+# scale of 0 to 1
+OI__VISION__PID__MAX_SPEED=0.25
+
+# whether there is a drivetrain on our robot
```

Comment:
Make this a TODO: remove during final integration. Also add TODO where it's used.

Reply:
Fixed